scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -327,7 +327,7 @@ def generate_command(self):
             self.source_dir, os.path.expanduser(os.path.expandvars(src)))) for src in pytest_root])
 
         if handler.options.pytest_args:
```

Comment:
Please move this if-statement to the end of this method to allow override the rest of parameters with `--pytest-args` from CLI.
e.g. to use `--log-level-cli=INFO` to not log DEBUGs from pytest.